ert_gui/simulation/detailed_progress.py

```diff
@@ -258,9 +263,24 @@ def update_data(self, jobs, iteration, realization):
         self.resizeColumnsToContents()
         self.model().modelReset.emit()
 
-        for file_name in self.open_files:
-            if self.open_files[file_name].isVisible():
-                self.open_files[file_name].reload(file_name)
+
+class FileUpdateThread(QThread):
+    text_read = pyqtSignal(str)
+
+    def __init__(self, file_name):
+        QThread.__init__(self)
+        self._file = open(file_name, 'r')
+        self.running = True
+
+    def run(self):
+        while self.running:
+            # Read at most 16KiB of data at a time so Python doesn't lock up
+            text = self._file.read(16384)
+            if len(text) > 0:
+                self.text_read.emit(text)
+            self.msleep(50)
```

Comment:
When this `self._file` is located on an NFS share, I think reading from it every `50` miliseconds will do _something_. We'd have to test, but what is wrong with 1 second? Some jobs run for hours…

Reply:
If we revert to every 1 seconds, I don't think we'd have to do testing with the runpath located on NFS.

If we want to poll a file every 50ms, and the users run Ert primarily with a runpath on NFS, we do need to test.

Reply:
50ms feels better locally, but I agree that we'll be hitting the NFS too often. I'll revert it to 1s.